### PR TITLE
NO-ISSUE: Fix dmn-dev-deployment-form-webapp dev server for windows

### PR DESCRIPTION
**Pull request by ljmotta**
## On this PR

 - Add shell parameter to the `spawn` function for windows compatibility;
 - Add specific env setup with `pnpm setup:env:win32` and `cross-env`;
 - Add missing env variable in `webpack.config.js`

### CODE REVIEW

**Review by tiagobento — APPROVED** [review]

File: packages/dmn-dev-deployment-form-webapp/package.json [hunk 1]
```diff
@@ -18,7 +18,13 @@
     "build:dev": "rimraf dist && webpack --env dev",
     "build:prod": "pnpm lint && pnpm test && rimraf dist && webpack",
     "lint": "run-script-if --bool \"$(build-env linters.run)\" --then \"kie-tools--eslint ./src\"",
-    "start": "DMN_DEV_DEPLOYMENTS_FORM_WEBAPP_QUARKUS_BASE_URL=http://localhost:$(build-env dmnDevDeploymentFormWebapp.dev.quarkusPort) node ./dev-webapp/start.js",
+    "powershell": "@powershell -NoProfile -ExecutionPolicy Unrestricted -Command",
+    "setup:env": "DMN_DEV_DEPLOYMENTS_FORM_WEBAPP_QUARKUS_BASE_URL=http://localhost:$(build-env dmnDevDeploymentFormWebapp.dev.quarkusPort)",
+    "setup:env:win32": "pnpm powershell \"cross-env DMN_DEV_DEPLOYMENTS_FORM_WEBAPP_QUARKUS_BASE_URL=http://localhost:$(build-env dmnDevDeploymentFormWebapp.dev.quarkusPort)\"",
+    "start": "pnpm run-script-os",
+    "start:darwin": "pnpm setup:env node ./dev-webapp/start.js",
+    "start:linux": "pnpm setup:env node ./dev-webapp/start.js",
```
Comment (thiagoelg) [inline packages/dmn-dev-deployment-form-webapp/package.json]:
```suggestion
    "start:darwin:linux": "pnpm setup:env node ./dev-webapp/start.js",
```

File: packages/dmn-dev-deployment-form-webapp/dev-webapp/start.js [hunk 2]
```diff
@@ -20,18 +20,22 @@ const { env } = require("../env");
...
+  [
+    "-f",
+    path.join(__dirname, "./quarkus-app"),
+    "clean",
+    "quarkus:dev",
+    "-Dmaven.test.skip",
+    `-Dquarkus.platform.version=${buildEnv.quarkusPlatform.version}`,
+    `-Dversion.org.kie.kogito=${buildEnv.kogitoRuntime.version}`,
+    `-Dquarkus.http.port=${buildEnv.dmnDevDeploymentFormWebapp.dev.quarkusPort}`,
+    `-Dkogito.service.url=http://localhost:${buildEnv.dmnDevDeploymentFormWebapp.dev.quarkusPort}`,
```
Comment (thiagoelg) [inline packages/dmn-dev-deployment-form-webapp/dev-webapp/start.js:34]: Should this be `DMN_DEV_DEPLOYMENTS_FORM_WEBAPP_QUARKUS_BASE_URL`?

File: packages/dmn-dev-deployment-form-webapp/package.json [hunk 3]
```diff
@@ -18,7 +18,13 @@
     "build:dev": "rimraf dist && webpack --env dev",
     "build:prod": "pnpm lint && pnpm test && rimraf dist && webpack",
     "lint": "run-script-if --bool \"$(build-env linters.run)\" --then \"kie-tools--eslint ./src\"",
-    "start": "DMN_DEV_DEPLOYMENTS_FORM_WEBAPP_QUARKUS_BASE_URL=http://localhost:$(build-env dmnDevDeploymentFormWebapp.dev.quarkusPort) node ./dev-webapp/start.js",
+    "powershell": "@powershell -NoProfile -ExecutionPolicy Unrestricted -Command",
+    "setup:env": "DMN_DEV_DEPLOYMENTS_FORM_WEBAPP_QUARKUS_BASE_URL=http://localhost:$(build-env dmnDevDeploymentFormWebapp.dev.quarkusPort)",
```
Comment (thiagoelg) [inline packages/dmn-dev-deployment-form-webapp/package.json]: I think this env var should be named `DMN_DEV_DEPLOYMENTS_FORM_WEBAPP__quarkusBaseUrl`, using the same pattern as other env vars.

Also, can we use the `baseUrl` from the `data.json` file?

File: packages/dmn-dev-deployment-form-webapp/dev-webapp/start.js (hunk 2 above)
Comment (ljmotta) [reply packages/dmn-dev-deployment-form-webapp/dev-webapp/start.js:34]: I could use the `dmnDevDeploymentFormWebapp.quarkusBaseUrl` but It wouldn't work if the user changed its value as the quarkus server will still run in the `quarkusPort`. I guess it's better to keep this way, as it's a dev weapp.

File: packages/dmn-dev-deployment-form-webapp/package.json (hunk 3 above)
Comment (ljmotta) [reply packages/dmn-dev-deployment-form-webapp/package.json]: I've removed the env var and now I'm using the `baseUrl`. Thanks!

**Pull request merged**